addons/web/static/src/core/commands/command_palette.variables.scss

```diff
@@ -0,0 +1,8 @@
+$o-command-palette-margin-top: 10vh;
+$o-command-palette-search-border: 1px solid rgba(0, 0, 0, 0.1);
+$o-command-palette-search-font-size: 1.25rem;
+$o-command-height: 2.8em;
+$o-command-palette-line-height: 1.8em;
+$o-command-palette-listbox-max-height: 50vh;
+$o-command-palette-kbd-shadow: 0px 1px 1px rgba(0, 0, 0, 0.2), inset 0px -1px 1px 1px rgba(230, 230, 230, 0.8), inset 0px 2px 0px 0px rgba(255, 255, 255, 0.8);
```

Comment:
Let's check with @Brieuc-brd if we can find a compromise and finally decide one single design for  kbd

Reply:
Checked with him, @Brieuc-brd added the kbd style there:
https://github.com/odoo/odoo/pull/84616 